src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+    @Override
+

```

Comment:
and also this one

Reply:
When i remove this empty line, checkstyle  tells me an error